applications/nrf_desktop/configuration/nrf52840dk_nrf52840/prj_wwcb.conf

```diff
@@ -41,7 +41,7 @@ CONFIG_DESKTOP_BLE_BOND_ENABLE=y
 CONFIG_CAF_BLE_ADV=y
 CONFIG_CAF_BLE_ADV_DIRECT_ADV=y
 CONFIG_CAF_BLE_ADV_FAST_ADV=y
-CONFIG_CAF_BLE_ADV_SWIFT_PAIR=y
+CONFIG_DESKTOP_BLE_ADV_SWIFT_PAIR=y
```

Comment:
since it was in caf somebody can actually use it - moving it to desktop will remove this feature from a common place

Reply:
The Swift Pair advertising data was actually defined in nRF Desktop specific def file (so it was not common data). Enabling the feature in CAF module required from user to manually add the data into his own def file (copy paste the data from nRF Desktop).

We should consider moving Swift Pair to separate file to simplify using it.